lib/modules/@apostrophecms/login/src/apos/components/TheApostropheLogin.vue

```diff
@@ -1,50 +1,118 @@
 <template>
-  <div class="apos-login">
-    <form @submit.prevent="submit">
-      <h2 v-if="error">{{ error }}</h2>
-      <fieldset>
-        <label>Username</label>
-        <input v-model="username" required />
-      </fieldset>
-      <fieldset>
-        <label>Password</label>
-        <input v-model="password" required type="password" />
-      </fieldset>
-      <fieldset>
-        <button type="submit">Log In</button>
-      </fieldset>
-    </form>
+  <div class="apos-login apos-theme-dark">
+    <div class="apos-login__overlay"></div>
+    <div class="apos-login__menu-overlay"></div>
+    <AposLoginBackground />
```

Comment:
Background image in data-uri format. Don't know if it is a good idea. I could have `import` a JPG file (works with webpack's `file-loader`).